frontend/src/features/projects/spl/forms/SurplusPropertyListForm.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React, { useState, useMemo } from 'react';
```

Comment:
I have a PR that I believe modifies this form to resolve issues with some workflows.